connectivity/lorawan/tests/UNITTESTS/features/lorawan/loraphyeu433/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set(TEST_NAME lorawan-loraphy-eu433-unittest)
+
+add_executable(${TEST_NAME})
+
+target_compile_definitions(${TEST_NAME}
+    PRIVATE
+        MBED_CONF_LORA_DOWNLINK_PREAMBLE_LENGTH=5        
+        MBED_CONF_LORA_TX_MAX_SIZE=255
+        MBED_CONF_LORA_UPLINK_PREAMBLE_LENGTH=8
+)
+
+target_sources(${TEST_NAME}
+    PRIVATE
+        ${MBED_PATH}/connectivity/lorawan/lorastack/phy/LoRaPHYEU433.cpp
+        Test_LoRaPHYEU433.cpp
+)
+
+target_link_libraries(${TEST_NAME}
+    PRIVATE
+        mbed-headers
+        mbed-stubs
+        mbed-stubs-headers
```

Comment:
I really don't understand why we need to pull in so many headers just for a single cpp file.